### PR TITLE
Update Python Test Files

### LINKED ISSUE #5501: Update Python Test Files
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Super-linter supports several Python linters and formatters. However, the contents of the test files differ between ruff and the others.

FYI: https://github.com/super-linter/super-linter/tree/e71a37d49d588b33a0d6e10f364a682e7704460e/test/linters

### Expected Behavior

For all Python tools, the same test files should be used. Moreover, they should be simple (meaning, it would be better to align them with the test files for ruff).

### Anything else?

Comment from ferrarimarco in #5456: https://github.com/super-linter/super-linter/pull/5456#issuecomment-2039055229, https://github.com/super-linter/super-linter/pull/5456#discussion_r1553245110

### PR DESCRIPTION
**Pull request by Okabe-Junya**
# Proposed changes

- align python test files to ruff.

fix #5501 

## Readiness checklist

In order to have this pull request merged, complete the following tasks.

### Pull request author tasks

- [ ] I included all the needed documentation for this change.
- [x] I provided the necessary tests.
- [x] I squashed all the commits into a single commit.
- [x] I followed the [Conventional Commit v1.0.0 spec](https://www.conventionalcommits.org/en/v1.0.0/).
- [ ] I wrote the necessary upgrade instructions in the [upgrade guide](../docs/upgrade-guide.md).
- [x] If this pull request is about and existing issue,
  I added the `Fix #ISSUE_NUMBER` label to the description of the pull request.

### Super-linter maintainer tasks

- [ ] Label as `breaking` if this change breaks compatibility with the previous released version.
- [ ] Label as either: `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`.

### CODE REVIEW

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
Hi @Okabe-Junya !

Thanks for this PR.

A small thing to check, so we can run the rest of the CI jobs.

File: test/linters/python_isort/python_bad_1.py [hunk 1]
```diff
@@ -1,195 +1,4 @@
-import json
-from os import getenv, path
-from pprint import pprint
-import sys
```
[the remark this answers is not in the document]
Comment (Okabe-Junya) [reply test/linters/python_isort/python_bad_1.py]: Oh, sorry for my missed operation.

isort checks the `import ...` or `from ... import ...` statements, so I shouldn't have removed this part. I have fixed it. It seems that the CI is passing now, except for the preview release note.

**Comment by Okabe-Junya** [comment]
Hi @ferrarimarco, thank you for your review!! And, sorry for my delay in response.

I checked your review comment and fixed it!! Please take a look again 💙 thanks!!

**Comment by ferrarimarco** [comment]
No worries, thanks for this work!

**Review by ferrarimarco — APPROVED** [review]

**Comment by Okabe-Junya** [comment]
Thank you for your kind supports!!

**Pull request merged**